test/alerts/alerts.sh

```diff
@@ -8,3 +8,44 @@ if [ $? -ne 0 ]; then
     echo "Checking of prometheus rules failed."
     exit 1
 fi
+
+helm init --client-only
+helm repo add bugroger-charts https://raw.githubusercontent.com/BugRoger/charts/repo
```

Comment:
do we still need this?  I thought this would be gone with the switch to stable grafana?

Reply:
needed until #439 is merged. removing repo in https://github.com/sapcc/kubernikus/pull/439/commits/f08882bdd00373288b27ce5e9805714b23c0f35f